lib/origen/site_config.rb

```diff
@@ -1,15 +1,14 @@
 module Origen
   class SiteConfig
+    # NOTE: Gems are not allowed to be required here, only Ruby stlibs
     require 'pathname'
     require 'yaml'
     require 'etc'
     require 'erb'
```

Comment:
@coreyeng, note that there were a couple of remaining gems in site_config.rb that I had to remove. I've replicated the red functionality locally, but I couldn't see httparty actually being used anywhere.
This does mean that if you want to add support for a network config file then you will need to use `net/http` rather than a gem to fetch it.